functions/go/search-replace/README.md

```diff
@@ -75,7 +85,9 @@ Alternatively, data can be passed as key-value pairs in the CLI
 $ kpt fn eval --image gcr.io/kpt-fn/search-replace:unstable -- 'by-path=metadata.name' 'put-value=the-deployment'
 ```
 
-Supported Path expressions:
+### Field path patterns
+
+`--by-path` matcher supports the following patterns:
```

Comment:
For consistency, these expressions should have both:

- a concise table (like you have below for file path)
- Examples like shown here.

Otherwise, it's inconsistent documentation.

Reply:
Updated both sections to be consistent.